### PR TITLE
feat(sample): create sensible samples

### PR DESCRIPTION
**Pull request by georgettica**
now you can 'make sample-install' and 'make sample-uninstall' to work with samples

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,6 +63,14 @@ deploy: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+# Install CRDs into a cluster
+sample-install: manifests kustomize
+	$(KUSTOMIZE) build config/samples | kubectl apply -f -
+#
+# Install CRDs into a cluster
```
Comment (RiRa12621) [inline Makefile:70]: Uninstall?

**Review by RiRa12621 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (georgettica) [reply Makefile:70]: Oh, copy pasta error.

Will.fix

File: config/samples/kustomization.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ## This file is auto-generated, do not modify ##
 resources:
+- monitoring_v1alpha1_clusterurlmonitor.yaml
```
Comment (RiRa12621) [inline config/samples/kustomization.yaml:3]: Why was that actually not present before?

Comment (georgettica) [reply config/samples/kustomization.yaml:3]: Nope, Manuel added it and it was not in the kustomization file (I guess he assumed it takes everything in the folder)

**Pull request merged**